apps/src/applab/AppLabView.jsx

```diff
@@ -64,19 +63,12 @@ var AppLabView = React.createClass({
         <ProtectedStatefulDiv
             id="codeWorkspace"
             renderContents={this.props.renderCodeWorkspace} />
-      </StudioAppWrapper>
+      </ConnectedStudioAppWrapper>
     );
-  },
-
-  shouldHideToggle: function () {
-    return this.props.isShareView || this.props.isDesignModeHidden;
-  },
-
-  shouldHideViewDataButton: function () {
-    return this.props.isViewDataButtonHidden ||
-        this.props.isDesignModeHidden ||
-        this.props.isShareView ||
-        !this.props.isEditingProject;
   }
 });
-module.exports = AppLabView;
+module.exports = connect(
+  (state) => ({
```

Comment:
Though this is a .jsx file, what you're writing here is still just Javascript :-P

It does seem like there's some silliness in not allowing it, when it works just fine, but I do think there's value in having an explicit point when we start using it (and have good linting, etc. around it). I do wonder some though if not getting to use ES6 will make redux feel somewhat less pleasant :(